src/components/checkoutpage.jsx

```diff
@@ -0,0 +1,297 @@
+import React, {Component} from 'react';
+import { Col, Button, Form, FormGroup, Label, Input, Row, Container, Breadcrumb, BreadcrumbItem} from 'reactstrap';
+import SectionHeadingAndWhiteLine from './complibrary/sectionheadingandwhiteline';
+import UpdateCart from './complibrary/updatecart';
+
+class CheckoutPage extends Component {
+
+    constructor(...args){
+        super(...args);
+        this.state = {};
+        this.onRadioBtnClick = this.onRadioBtnClick.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onRadioBtnClick(rSelected) {
+        var shippingMethodAndPrice={};
+        this.setState({ rSelected });
+        console.log("shipping method selected :" + rSelected);
+        //Set the shipping method and price in the cart object.
+        switch(rSelected.toString()){
+            case '1': shippingMethodAndPrice ={"shippingMethod":"Economy Shipping","shippingPrice":"0.00", "expectedDelivery":" 3-4 weeks"};
+            break;
+
+            case '2': shippingMethodAndPrice ={"shippingMethod":"Standard Shipping","shippingPrice":"7.99", "expectedDelivery":" 1 weeks"};
+            break;
+
+            case '3': shippingMethodAndPrice ={"shippingMethod":"Expedite Shipping","shippingPrice":"24.95", "expectedDelivery":" Next day by air"};
+            break;
+
+            default:
+            break;
+        }
+        //UpdatedCart price in local storage
+        UpdateCart("updateShippingMethodAndPrice", shippingMethodAndPrice);
```

Comment:
if it's a function that's called as a function, it's helpful naming in `camelCase`